src/nprintml/label/aggregator/base.py

```diff
@@ -39,8 +39,26 @@ def load_label(self, labels_csv):
         """Load labels, which are expected to be in item, label column
         format where item is the index.
 
+        A header line "item,label" (case-insensitive) is optional.
+
+        For example:
+
+            item,label
+            key1,label1
+            key2,label2
+            key3,label3
+
         """
-        return pd.read_csv(labels_csv, index_col=0)
+        # enforce column names (item, label)
+        label = pd.read_csv(labels_csv, index_col='item', names=('item', 'label'))
+
+        # check for optional header line
+        if not label.empty:
+            row0 = label.iloc[0]
+            if row0.name.lower() == 'item' and row0.label.lower() == 'label':
+                return label.iloc[1:]
+
+        return label
```

Comment:
We had documented that there should be a header with the titlecased column `Label`, but were making reference to the column `label` in the code. So I made the code set the column names it expects, and made specification of the header optional for the user.